include/net/nrf_cloud_agps.h

```diff
@@ -30,7 +30,7 @@ extern "C" {
  *
  * @return 0 if successful, otherwise a (negative) error code.
  */
-int nrf_cloud_agps_request(const struct gps_agps_request request);
+int nrf_cloud_agps_request(const struct nrf_modem_gnss_agps_data_frame *request);
```

Comment:
This is a much welcomed change, but you'll need to update `nrf\applications\serial_lte_modem\src\gnss\slm_at_gnss.c` as well

Reply:
SLM is already updated in this PR. Everything is in the same commit, I guess it could have been better to split it in several commits for better visibility.